corehq/apps/cloudcare/static/cloudcare/js/form_entry/entries.js

```diff
@@ -523,6 +523,32 @@ hqDefine("cloudcare/js/form_entry/entries", function () {
         self.rawAnswer(constants.NO_ANSWER);
     };
 
+    /**
+     * Represents a single button that cycles through choices
+     */
+    function ButtonSelectEntry(question, options) {
+        var self = this;
```

Comment:
It could even be `const` since it's not getting redefined, but I actually prefer `var` here because that's how other entry classes are set up. There's not a _ton_ of consistency of variable types in that file, but I also think it would be out of scope here to redefine them all in this PR given it's such a key piece of web apps.